packages/commerce-sdk-react/src/hooks/Test/config.ts

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2023, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import {ShopperBasketsTypes, ShopperCustomersTypes} from 'commerce-sdk-isomorphic'
+import {
+    ApiClients,
+    ApiOptions,
+    CacheUpdate,
+    CacheUpdateMatrix,
+    CacheUpdateUpdate,
+    MergedOptions
+} from '../types'
+import {and, matchesApiConfig, pathStartsWith} from '../utils'
+
+type Client = ApiClients['shopperBaskets']
+type Basket = ShopperBasketsTypes.Basket
+type CustomerBasketsResult = ShopperCustomersTypes.BasketsResult
+type BasketOptions = MergedOptions<Client, ApiOptions<Basket>>
+
+const updateBasketQuery = (
+    customerId: string | null,
+    basketId: string | undefined,
+    newBasket: Basket
+): Pick<CacheUpdate, 'update'> => {
+    if (!basketId) return {}
+    const update: Array<CacheUpdateUpdate<Basket> | CacheUpdateUpdate<CustomerBasketsResult>> = [
+        {
+            queryKey: ['/baskets', basketId, {basketId}],
+            updater: newBasket
+        }
+    ]
+    if (customerId) {
+        const updateCustomerBaskets: CacheUpdateUpdate<CustomerBasketsResult> = {
+            // Since we use baskets from customer basket query, we need to update it for any basket mutation
+            queryKey: ['/customers', customerId, '/baskets', {customerId}],
+            updater: (oldData) => {
+                // do not update if response basket is not part of existing customer baskets
+                if (!oldData?.baskets?.some((basket) => basket.basketId === basketId)) {
+                    return undefined
+                }
+                const updatedBaskets = oldData.baskets.map((basket) => {
+                    return basket.basketId === basketId ? newBasket : basket
+                })
+                return {
+                    ...oldData,
+                    // TODO: Remove type assertion when RAML specs match
+                    baskets: updatedBaskets as CustomerBasketsResult['baskets']
```

Comment:
Shopper Baskets and Shopper Customers use different definitions for `Basket` that are like 99% similar, but don't exactly match. 😐